examples/rendercontrol_opengl/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Core Gui OpenGL Qml Quick)
+
+set(SOURCES
+    rendercontrol.qrc
+    main.cpp
+    window_singlethreaded.cpp
+)
+
+qt_add_executable(rendercontrol_openglexample WIN32 ${SOURCES})
+
+target_link_libraries(rendercontrol_openglexample PRIVATE
+    Qt${QT_VERSION_MAJOR}::Core
+	Qt${QT_VERSION_MAJOR}::Gui
```

Comment:
Better indent is needed.

Reply:
Fixed in 6567229af417e377a3ff249ca376221a99b394c3.